vuepress/docs/next/docs/reference/local-tips-and-tricks.md

```diff
@@ -63,23 +63,21 @@ The namespace will be recreated, preserving the images already pulled, so it's u
 We recommend using Multipass to quickly spin up an Ubuntu VM to host a local Kubernetes cluster for test purposes. A local environment is often useful, but most teams utilize a shared Kubernetes cluster. This shared cluster is managed by an operations team, and installed either on premise or with a cloud provider for full integration testing, CI/CD, DevOps, etc. 
 
 ## Network Issues
-A local Entando 7 quickstart installation (e.g. what you'll get if you follow the [Getting Started](../../docs/getting-started/) guide) may use a set of local domain names to enable access to Entando services. Your IP address will vary, but may look something like this:
+A local Entando 7 quickstart installation (e.g. what you get if you follow the [Getting Started](../../docs/getting-started/) guide) may use a set of local domain names to enable access to Entando services. Your IP address will vary, but may look something like this:
 ```
-quickstart-entando.192.168.99.1.nip.io
-quickstart-kc-entando.192.168.99.1.nip.io
-quickstart-eci-entando.192.168.99.1.nip.io
+quickstart.192.168.99.1.nip.io
+YOUR-APP.192.168.99.1.nip.io
 ```
-The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
+The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando-app.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments, there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
```

Comment:
We don't have the ENTANDO_DEFAULT_ROUTING_SUFFIX anymore, maybe just change to this: 
...`ingressHostname` (e.g. in your entando.yaml) is based on an IP address that is created during the initial VM installation.